android/external/MakefileTemplate

```diff
@@ -257,13 +257,27 @@ nettle-install: $(LOCAL)/lib/libnettle.so
 # BOTAN2
 
 botan.src.stamp: ../downloads/botan.src.stamp
-	cp -r ../downloads/botan botan
+	cp -r $(BOTAN_SRC) botan
 	touch $@
 
-botan/Makefile: botan.src.stamp
+## Build jitter
+jitter-build: $(LOCAL)/lib/libjitterentropy.a
+
+$(LOCAL)/lib/libjitterentropy.a:
+	$(MAKE) -C $(JITTER_SRC) jitterentropy-static
+	$(MAKE) -C $(JITTER_SRC) jitterentropy-static install-includes PREFIX=$(LOCAL)
+	$(MAKE) -C $(JITTER_SRC) jitterentropy-static install-static PREFIX=$(LOCAL)
+
+## Clean jitter
+jitter-clean:
+	-$(MAKE) -C $(JITTER_SRC) clean
+
+botan/Makefile: botan.src.stamp $(LOCAL)/lib/libjitterentropy.a
```

Comment:
Those stamp files always looked like an ugly hack, and I can't imagine why they would be needed. You patch botan sources, you either rebuild it completely or you rely on its generated build system (Makefile). Or you remove the Makefile to trigger a reconfigure as well, if needed. But up to you.

Reply:
hum when we run the build for all 4 archs then we would need to also remove the Makefile created by previous arch build to avoid issues. This sounds bad as the Makefile will have to be created every single time but it shouldn't?

Reply:
What I can do with no issues is removing the .stamp file for botan I think.

Reply:
Please let me know your thoughts.

Reply:
used botan/Makefile as stamp.